### PR TITLE
Add type name for FixtureLongId

### PR DESCRIPTION
**Pull request by matthiaso**

### CODE REVIEW

**Review by paolobazzi — APPROVED** [review]

File: org.eclipse.scout.rt.dataobject.test/src/test/java/org/eclipse/scout/rt/dataobject/id/IdInventoryTest.java [hunk 1]
```diff
@@ -35,14 +36,14 @@ public void testGetIdClass() {
   @Test
   public void testGetTypeNameByClass() {
     assertEquals("scout.FixtureUuId", m_inventory.getTypeName(FixtureUuId.class));
-    assertNull(m_inventory.getTypeName(FixtureLongId.class));
+    assertNull(m_inventory.getTypeName(FixtureLongIdWithoutTypeName.class));
```
Comment (paolobazzi) [inline org.eclipse.scout.rt.dataobject.test/src/test/java/org/eclipse/scout/rt/dataobject/id/IdInventoryTest.java:40]: you could add another testcase for FixtureLongId

File: org.eclipse.scout.rt.dataobject.test/src/test/java/org/eclipse/scout/rt/dataobject/id/IdInventoryTest.java [hunk 2]
```diff
@@ -35,14 +36,14 @@ public void testGetIdClass() {
...
     assertNull(m_inventory.getTypeName((Class<? extends IId>) null));
   }
 
   @Test
   public void testGetTypeNameByInstance() {
     assertEquals("scout.FixtureUuId", m_inventory.getTypeName(FixtureUuId.of("3c5a66be-12b4-45c6-9e59-8c31cf92dcfb")));
-    assertNull(m_inventory.getTypeName(FixtureLongId.of(100L)));
+    assertNull(m_inventory.getTypeName(FixtureLongIdWithoutTypeName.of(100L)));
```
Comment (paolobazzi) [inline org.eclipse.scout.rt.dataobject.test/src/test/java/org/eclipse/scout/rt/dataobject/id/IdInventoryTest.java:48]: you could add another testcase for FixtureLongId

**Review by paolobazzi — APPROVED** [review]

File: org.eclipse.scout.rt.dataobject.test/src/test/java/org/eclipse/scout/rt/dataobject/id/IdInventoryTest.java (hunk 1 above)
Comment (matthiaso) [reply org.eclipse.scout.rt.dataobject.test/src/test/java/org/eclipse/scout/rt/dataobject/id/IdInventoryTest.java:40]: done

File: org.eclipse.scout.rt.dataobject.test/src/test/java/org/eclipse/scout/rt/dataobject/id/IdInventoryTest.java (hunk 2 above)
Comment (matthiaso) [reply org.eclipse.scout.rt.dataobject.test/src/test/java/org/eclipse/scout/rt/dataobject/id/IdInventoryTest.java:48]: done

**Comment by matthiaso** [comment]
One failing test also failed on continuous build

**Pull request merged**